### PR TITLE
Add more scrap whitelist to toolbox

### PR DESCRIPTION
**Pull request by Hmeister-real**
## About The Pull Request
Someone told me wyci when i asked in ooc, so i did! Adds more whitelist for scrap and other materials to the toolbox, Scavs rejoice!

## Why It's Good For The Game
Makes toolbox more useful, alternative to carrying a second backpack for scrap

## Changelog
:cl: Hmeister
qol: Toolboxes now have a less strict whitelist and hold most scraps and junk.

### CODE REVIEW

**Comment by Hmeister-real** [comment]
idk why it's failing, nothings wrong. probably a MS Git thing

**Comment by ProfessorPopoff** [comment]
Yeah- checks are a bit wonky right now.

We actually do have some errors in regards to the toolboxes. I don't have them handy right now but there's runtimes on init. 

That being said, I think better implementation of this would actually be removing the whitelist entirely and just letting people use the container for what it is- a container.

The only container i'd really like to continue this way is maybe the doctor's bag/medical kits because those are usually pretty specifically built inside with all sorts of pockets and straps and whatnot- Plus it wouldn't hurt to have /some/ specialty.

Though on a more soul-bound train of thought: Tools are only half of what goes into my toolbox IRL lol. 

tldr: Nuke toolbox whitelists

**Comment by Hmeister-real** [comment]
> tldr: Nuke toolbox whitelists

thought that would have been a little too drastic, but looking in retrospect would make more sense, il close this and make a different PR to nuke a few whitelists
